libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/Edge.kt

```diff
@@ -0,0 +1,157 @@
+package com.mapbox.navigation.core.trip.model.eh
+
+import com.mapbox.geojson.LineString
+
+/**
+ * Basic Edge class
+ *
+ * @param id the Edge identifier
+ * @param level the level of the Edge (0 being the mpp, 1 branches of the mpp,
+ * 2 branches of level 1 branches, etc)
+ * @param probability the probability of the Edge in percentage
+ * @param heading the bearing in degrees clockwise at the start of the edge
+ * @param length the Edge's length in meters
+ * @param out the outgoing Edges
+ * @param parent the parent Edge
+ * @param frc the edge's functional road class
+ * @param speed the Edge's average speed
+ * @param ramp is the edge a ramp?
+ * @param motorway is the edge a motorway?
+ * @param bridge is the edge a bridge?
+ * @param tunnel is the edge a tunnel?
+ * @param toll is the edge a toll road?
+ * @param names the edge's names
+ * @param curvature the edge's curvature
+ * @param geometry optional geometry if requested
+ * @param speedLimit the Edge's max speed
+ * @param laneCount the edge's lane counts
+ * @param meanElevation the edge's mean elevation
+ * @param countryCode the edge's country code
+ * @param stateCode the edge's state code
+ */
+class Edge internal constructor(
+    val id: Long,
+    val level: Byte,
+    val probability: Double,
+    val heading: Double,
+    val length: Double,
+    val out: List<Edge>,
+    var parent: Edge?,
+    val frc: String,
```

Comment:
what does frc stand for? can we add it to the documentation? i'm also in favor of renaming this variable to be what it is.

Reply:
It's Functional Road Class.